uavcan/equipment/device/1100.Temperature.uavcan

```diff
@@ -0,0 +1,11 @@
+#
+# Generic device temperature
+#
+
+uint16_t device_id
+
+float16 temperature
+
+uint8 ERROR_FLAG_OVERTEMPERATURE = 1
+uint8 ERROR_FLAG UNDERTEMPERATURE = 2
```

Comment:
My English is rusty so I could be wrong, but I suspect that better terms would be "overheating" and ["overcooling"](https://en.wiktionary.org/wiki/overcooling).